admin/class-admin-init.php

```diff
@@ -613,6 +613,10 @@ public function show_hook_deprecation_warnings() {
 				'version'     => '5.8',
 				'alternative' => 'wpseo_breadcrumb_single_link_info',
 			),
+			'wpseo_stopwords'              => array(
+				'version'     => '6.3',
+				'alternative' => 'none',
```

Comment:
I would set this to `null`. WordPress will render an other message if the replacement value is `null`